redisinsight/ui/src/constants/pages.ts

```diff
@@ -49,4 +50,6 @@ export const Pages = {
     `/${instanceId}/${PageNames.triggeredFunctions}/${PageNames.triggeredFunctionsLibraries}`,
   triggeredFunctionsFunctions: (instanceId: string) =>
     `/${instanceId}/${PageNames.triggeredFunctions}/${PageNames.triggeredFunctionsFunctions}`,
+  rdi: `${rdi}`,
+  rdiEditInstance: (instanceId: string) => `${rdi}?editInstance=${instanceId}`
```

Comment:
I guess will be better replace instanceId with rdiInstanceId. It will be more clear

Reply:
It feels redundant to have `rdiInstanceId` under a url that has `rdi/` as its base. Is the confusion around `instanceId` being reserved for a `databaseId` or something to that effect?